build/Dockerfile

```diff
@@ -9,14 +7,21 @@ COPY go.sum .
 
 RUN go mod download
 
-COPY configs configs/
-COPY cmd cmd/
-COPY internal internal/
-COPY pkg pkg/
-COPY web web/
+COPY . ./
 
-RUN go build -o /server ./cmd/server/*.go
-RUN go build -o /database ./cmd/database/*.go
+RUN CGO_ENABLED=0 go build -ldflags="-w -s" -o /server ./cmd/server/*.go \
+ && CGO_ENABLED=0 go build -ldflags="-w -s" -o /database ./cmd/database/*.go
```

Comment:
What purpose of CGO_ENABLED=0 and -ldflag?

Reply:
CGO_ENABLED=0 means that the resulting binary will not be linked to any C libraries.
You will get the smallest binaries if you compile with -ldflags '-w -s'.
-s option disable symbol table and  -w option disable DWARF generation.